cypress/components/icon/icon.cy.tsx

```diff
@@ -181,23 +199,23 @@ context("Tests for Icon component", () => {
     );
 
     it.each([
-      ["left", "bottom", "top"],
-      ["top", "bottom", "top"],
-      ["left", "top", "bottom"],
-      ["bottom", "top", "bottom"],
-      ["bottom", "left", "bottom"],
-      ["bottom", "right", "bottom"],
-      ["top", "left", "top"],
-      ["top", "right", "top"],
-      ["right", "bottom", "right"],
-      ["right", "top", "right"],
-    ])(
+      [["left"], "left", "bottom", "top"],
```

Comment:
why this needs to be an `array` in `array`?

Reply:
The `tooltipFlipOverrides` prop is of type `TooltipPositions[]`, so we are expecting consumers to be passing in an array of position strings here